addons/resource.language.en_gb/resources/strings.po

```diff
@@ -17814,7 +17820,29 @@ msgctxt "#36614"
 msgid "Show add-ons that are currently running in the background."
 msgstr ""
 
-#empty strings from id 36615 to 36899
+#: system/settings/settings.xml
```

Comment:
well, I tried to give it a "generic" description, but feel free to rephrase. And thanks for adding - all good from my end.